### PR TITLE
Add greencart disclaimer

### PR DESCRIPTION
**Pull request by rodolfopietro97**
Hi Team,

Following the recent discussion and the similarity observed with the [Greencart DAPP](https://www.greencart.vet) use case, we propose adding a disclaimer to this repository. This disclaimer will help clarify the intended use and limitations of the content within the repo.

Thank you for your consideration.

@Valazan @fabiorigam @pierobassa @vineet-codes @bacco1977

### CODE REVIEW

**Comment by pierobassa** [comment]
image

Disclaimer should be coherent with the max width and centered.

File: apps/frontend/src/App.tsx [hunk 1]
```diff
@@ -45,7 +54,11 @@ function App() {
             </Container>
           </Flex>
           <Footer />
-
+          <Alert status='warning'>
+            <AlertIcon />
+            <AlertTitle>Disclaimer</AlertTitle>
+            <AlertDescription>The use case of this template is inspired by the <Link color='teal.500' className="hover:underline text-blue-500" href={"https://www.greencart.vet/"}>GreenCart Dapp</Link>. The code has been built from scratch and does not contain any references to the GreenCart codebase.</AlertDescription>
```
Comment (pierobassa) [inline apps/frontend/src/App.tsx:60]: tailwind is not used

Comment (rodolfopietro97) [reply apps/frontend/src/App.tsx]: Didn't see 😅. Corrected: https://github.com/vechain/x-app-template/pull/12/commits/d24fd89209a22542ee078de948350a6b60857376

File: apps/frontend/src/App.tsx [hunk 2]
```diff
@@ -10,6 +19,7 @@ import {
 } from "./components";
 import { lightTheme } from "./theme";
 import { GoogleReCaptchaProvider } from "react-google-recaptcha-v3";
+import { Disclaimer } from "./components/Disclaimer/Disclaimer.tsx";
```
Comment (pierobassa) [inline apps/frontend/src/App.tsx]: should import from `./components`

Comment (rodolfopietro97) [reply apps/frontend/src/App.tsx]: Done here https://github.com/vechain/x-app-template/pull/12/commits/467c9510d0d480a83abf34fdc201ce3f745a705c

**Review by pierobassa — APPROVED** [review]

**Review by Agilulfo1820 — APPROVED** [review]

**Review by pierobassa — CHANGES_REQUESTED** [review]
Some changes to be made as suggested by @vineet-codes which make sense

- Let's not use "Disclaimer" as wording but "Note". 
- Inspiration should not be only Greencart but Mugshot, Cleanify & Greencart as the template does inspire to all 3.

**Comment by Agilulfo1820** [comment]
Can we style this disclaimer differently, maybe part of the footer? It seems more like an ad than a disclaimer

**Review by akanoce — COMMENTED** [review]
I'm personally NOT in favour of merging this.
As @pierobassa pointed out, all the apps are kinda inspired by each other, I find very silly to add such a claim, as it does not benefit the users of this app/repo in any way.

While it's true that the use case it's kinda similar, as @rodolfopietro97 stated this have been built without looking at greencart source code, hence i don't see any issue publishing this without any note or disclaimer.

If we follow this line, all the search engines should quote google, all the browsers should cite WorldWideWeb and so on. There is no such thing as a copyright on an use case or idea. Another thing is violation of copyright by copying the codebase, but this is clearly not the case.

**Comment by grenos** [comment]
I agrre with @akanoce. Such a banner doesn't add any value whatsoever to the template.

**Comment by pierobassa** [comment]
Closing for stale over 2 weeks. Please re-open another PR if you deem it necessary

**Pull request closed without merging**